### PR TITLE
Adds linter rules for license and import. Fixes linting (#319)

### PR DESCRIPTION
**Pull request by tianleh**
Signed-off-by: Ashwin Pc <ashwinpc@amazon.com>

Signed-off-by: Ashwin Pc <ashwinpc@amazon.com>
(cherry picked from commit 317f15665b900737565eff8c8b0b8534f47ddac1)

### Description

[Describe what this change achieves]

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by tianleh** [comment]
This is a backport PR originated from https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/319

File: site/js/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```
Comment (seraphjiang) [inline site/js/dashboard.js:5]: adding comments?, what is this file for?

File: cypress/utils/plugins/notifications-dashboards/constants.js [hunk 2]
```diff
@@ -4,4 +4,4 @@
  */
 
 export const NOTIFICATIONS_PLUGIN_NAME = 'notifications-dashboards';
-export const delay = 1000;
+export const NOTIFICATIONS_DELAY = 1000;
```
Comment (seraphjiang) [inline cypress/utils/plugins/notifications-dashboards/constants.js:7]: delay millisecond or second?

**Review by CCongWang — APPROVED** [review]

**Review by seraphjiang — APPROVED** [review]

File: site/js/dashboard.js (hunk 1 above)
Comment (tianleh) [reply site/js/dashboard.js:5]: license header which is used in OpenSearch development

File: cypress/utils/plugins/notifications-dashboards/constants.js (hunk 2 above)
Comment (tianleh) [reply cypress/utils/plugins/notifications-dashboards/constants.js:7]: milliseconds https://docs.cypress.io/api/commands/wait#Syntax

**Pull request merged**